### DISCUSSION TITLE
How to download Serverpod Insights?

### DISCUSSION THREAD

**Discussion by Pingear** [serverpod/serverpod#594] [Q&A] [answered]
Does anyone know where to download the Serverpod Insights companion app? The link in the documentation leads to an error page
And where is the repository with the source code of this application?

<img width="772" alt="image" src="https://user-images.githubusercontent.com/16469484/215583478-40c97271-cfa3-4e8b-a5b5-51954290bf1b.png">

**Comment by yahu1031** [comment, marked as answer]
[Here](https://serverpod.dev/insights/Serverpod-1.0.0.zip)

**Reply by yahu1031** [reply]
In the link, edit it to `.zip` - ~~`-zip`~~

**Reply by Pingear** [reply]
Thank you) I also already guessed to change the dash to a dot in the address))
What about the source code, is the admin panel also an open source application?

**Reply by yahu1031** [reply]
Not as of now.

**Reply by Pingear** [reply]
Do you know why?

**Reply by Pingear** [reply]
And when)?

**Reply by yahu1031** [reply]
Not sure but I think the app is pretty decent and it doesn't need any updates. Incase it needs, then may be SQL query runner, etc can be added (@vlidholt), And may be it won't be open-sourced as of now.

**Reply by Pingear** [reply]
This is little disappointing, in my opinion this application is an advantage of this server, I have been waiting for it for the last half a year)) And to be honest, I hoped that the application was also written using dart lang)

**Reply by vlidholt** [reply]
@Pingear Serverpod insights is written in Flutter/Dart. It's not open source as of now, but may be in the future.

**Comment by Pingear** [comment]
@vlidholt
Victor, I have a question, will the admin panel application be open source? Or you plan to charge for its use in the future.
We are considering your very promising server as a server for our future projects, so it is important to know such nuances in advance)

**Reply by vlidholt** [reply]
The basic features will always be free, we may be adding paid features to Serverpod Insights in the future (like analytics maybe).

**Reply by Pingear** [reply]
Is there documentation somewhere that describes the protocols for interaction between the admin application and the server?
It turns out that a third-party application will communicate with our server, which has access to our users' information.
How to make sure that the data on the server will not be compromised?

**Reply by yahu1031** [reply]
Unless you don't expose your connection details, Your connections will be safe.

**Reply by vlidholt** [reply]
The API between the application and the server is open-source. Everything communicated is private to the app and encrypted. No data will leave your computer (other than your email address when signing in). The key to Serverpod insights is stored in you `config/passwords.yaml` file. It is important to keep that one safe.

This is the API: https://github.com/serverpod/serverpod/blob/main/packages/serverpod/lib/src/endpoints/insights.dart
